<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="StackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Menu">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>160</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="createProfileButton">
         <property name="text">
          <string>Create Profile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loginButton">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>158</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>RaDoTech</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CreateProfile">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>381</width>
        <height>340</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="userInfo">
       <item row="6" column="2">
        <widget class="QLineEdit" name="countryText">
         <property name="placeholderText">
          <string>Your country</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="heightLbl">
         <property name="text">
          <string>Height, cm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lNameLbl">
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="countryLbl">
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="emailText">
         <property name="placeholderText">
          <string>Email Address</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLineEdit" name="confirmPasswordText">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>12345</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lastNameText">
         <property name="placeholderText">
          <string>Last name</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="birthdayLbl">
         <property name="text">
          <string>Date of Birth</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="firstNameText">
         <property name="placeholderText">
          <string>First name</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="emailLbl">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QDateEdit" name="birthdayEdit">
         <property name="displayFormat">
          <string>MM/dd/yyyy</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="phoneLbl">
         <property name="text">
          <string>Phone Number, +</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="genderLbl">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="passwordLbl">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="confirmPasswordLbl">
         <property name="text">
          <string>Confirm password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <layout class="QHBoxLayout" name="genderOptions">
         <item>
          <widget class="QRadioButton" name="maleButton">
           <property name="text">
            <string>Male</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="femaleButton">
           <property name="text">
            <string>Female</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="otherButton">
           <property name="text">
            <string>Other</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="weightLbl">
         <property name="text">
          <string>Weight, kg</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="passwordText">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>12345</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="fNameLbl">
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="phoneText">
         <property name="placeholderText">
          <string>12223334444</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="weightBox">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="heightBox">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>450</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Save and Continue</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginProfile">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>158</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>RaDoTech</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>230</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="emailLoginText">
         <property name="placeholderText">
          <string>Login (email)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLoginText">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="enterButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>350</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="deleteme"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
